source/contributing.rst

```diff
@@ -167,22 +173,22 @@ report using the guidelines in the :ref:`bug-reports` section.
 
 If you've done this but want to get contributing right away, you can take
 advantage of the fact that all Hyper projects use a continuous integration
-system. This will automatically run the tests against any pull request raised
-against any Hyper repository. The continuous integration system treats a
-regression in code coverage as a failure of the test suite.
+system. This system will automatically run the tests against any pull request
+raised against any Hyper repository. The continuous integration system treats
+a regression in code coverage as a failure of the test suite.
 
-Before a contribution is merged it must have a green run through the CI system.
+Before a contribution is merged, it must have a green run through the CI system.
 
 .. _tox: http://tox.readthedocs.org/en/latest/
 
 Code Review
 ~~~~~~~~~~~
 
 Contributions will not be merged until they've been code reviewed. You should
-implement any code review feedback unless you strongly object to it. In the
-event that you object to the code review feedback, you should make your case
-clearly and calmly. If, after doing so, the feedback is judged to still apply,
-you must either apply the feedback or withdraw your contribution.
+implement any code review feedback unless you strongly object to it. If
+you object to the code review feedback, you should make your case
+clearly and calmly. If, after doing so, the feedback is judged to apply still,
+you must either incorporate the feedback or withdraw your contribution.
```

Comment:
I think the "feedback is judged to apply still" sentence should just be rewritten, it was confusing the first time I read it and it's still confusing to read.